storage/client_test.go

```diff
@@ -58,9 +59,9 @@ func createTestStore(t *testing.T) *storage.Store {
 func createTestStoreWithEngine(t *testing.T, eng engine.Engine, clock *hlc.Clock,
 	bootstrap bool) *storage.Store {
 	rpcContext := rpc.NewContext(hlc.NewClock(hlc.UnixNano), rpc.LoadInsecureTLSConfig())
-	g := gossip.New(rpcContext)
+	g := gossip.New(rpcContext, 10*time.Millisecond, "")
```

Comment:
I've seen these constants sprinkled pretty liberally in test code. I think it makes sense to add a const for gossip.TestInterval.